dspace/src/main/docker-compose/db.entities.yml

```diff
@@ -0,0 +1,16 @@
+#
+# The contents of this file are subject to the license and copyright
+# detailed in the LICENSE and NOTICE files at the root of the source
+# tree and available online at
+#
+# http://www.dspace.org/license/
+#
+
+version: "3.7"
+
+services:
+  dspacedb:
+    image: dspace/dspace-postgres-pgcrypto:loadsql
```

Comment:
Could we add a small comment to describe the "loadsql" tag, and the fact that this uses the `dspace-postgres-pgcrypto-curl` source file.